packages/shared/src/components/profile/UserShortInfo.tsx

```diff
@@ -162,7 +162,7 @@ const UserShortInfoComponent = <Tag extends React.ElementType>(
           feedId={feedId}
         />
       )}
-      {!!showAward && !!user.award && (
+      {!!showAward && !!user.award && !!user.awardTransaction && (
```

Comment:
We still need to check for `user.award`?

Reply:
yeah, we still use image and name from there

Reply:
👍